packaging/macOS/freeze_program.py

```diff
@@ -39,6 +44,12 @@ def main(
     include_parsec_ext: Optional[Path] = None,
     wheel_it_dir: Optional[Path] = None,
 ):
+    log = logger.getChild(main.__name__)
+
+    log.info(f"BUILD_DIR={BUILD_DIR}")
+    log.info(
+        f"program_source={program_source}, include_parsec_ext={include_parsec_ext}, wheel_it_dir={wheel_it_dir}"
```

Comment:
It was easier for me, to detect what was printed from the program and the output of the sub commands
because the lines would stand-out from the others